wasmtest.sh

```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
+
+# wasmtest.sh is a shell script that can compile and run wasm tests.
+# There are three funtions for wasmteest.sh(1.test single file 2.test all files 3.test all tests from the file user give)
+# To test single file use: ./wasmtest.sh single <your file name>.c 
+# To compile single file use: ./wasmtest.sh singlecompile <your file name>.c
+# To run single file use: ./wasmtest.sh singlecompile <your file name>.c
+# To test all files use: ./wasmtest.sh all
+# To compile all files use: ./wasmtest.sh allcompile
+# To run all files use: ./wasmtest.sh allrun
+# To test all tests from the file user give use: ./wasmtest.sh files
+# To compile all tests from the file user give use: ./wasmtest.sh filescompile
+# To run all tests from the file user give use: ./wasmtest.sh filesrun
+
+glibc_base="/home/lind-wasm/src/glibc"
+wasmtime_base="/home/lind-wasm/src/wasmtime"
+
+CC="${CLANG:=/home/lind-wasm/clang+llvm-16.0.4-x86_64-linux-gnu-ubuntu-22.04}/bin/clang"
+
+export_cmd="export LD_LIBRARY_PATH=$wasmtime_base/crates/rustposix:\$LD_LIBRARY_PATH"
+
+compile_test_cmd_fork_test="$CC -pthread --target=wasm32-unknown-wasi --sysroot $glibc_base/sysroot -Wl,--import-memory,--export-memory,--max-memory=67108864,--export="__stack_pointer" [input] -g -O0 -o [output] && wasm-opt --asyncify --debuginfo [output] -o [output]"
```

Comment:
Change this to `compile_test_cmd_fork_test="$CC -pthread --target=wasm32-unknown-wasi --sysroot $glibc_base/sysroot -Wl,--import-memory,--export-memory,--max-memory=67108864,--export="__stack_pointer",--export=__stack_low [input] -g -O0 -o [output] && wasm-opt --asyncify --debuginfo [output] -o [output]"`
The `--export=__stack_low` added here is from current pthread fix PR(https://github.com/Lind-Project/lind-wasm/pull/67). Let's also add it for this file in this PR.